lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -1099,6 +1094,16 @@ def test_key_and_dates(self, query_as):
                 "dataset.owner:desc",
                 ["test", "fio_2", "uperf_1", "uperf_2", "uperf_3", "uperf_4", "fio_1"],
             ),
+            (
+                # Sort by stringified integer
+                "global.test.mcguffin:asc:str",
+                ["uperf_4", "uperf_2", "uperf_1", "test", "fio_2", "fio_1", "uperf_3"],
+            ),
+            (
+                # Sort by integer
+                "global.test.mcguffin:asc:int",
+                ["uperf_4", "uperf_3", "uperf_2", "uperf_1", "test", "fio_2", "fio_1"],
+            ),
```

Comment:
This result looks like descending order by name...it would be a better test if the result were unique to this query.

Reply:
Since the datasets aren't created/stored in that order, it *is* unique, and the dataset names aren't relevant to the sort anyway.

Reply:
> the dataset names aren't relevant to the sort anyway.

Well, I would _hope_ so...but...I'm not sure that the test can tell....  🙃